undertow/src/main/java/org/wildfly/extension/undertow/ListenerService.java

```diff
@@ -139,7 +140,15 @@ public void start(StartContext context) throws StartException {
             registerBinding();
         } catch (IOException e) {
             cleanFailedStart();
-            throw new StartException("Could not start http listener", e);
+            if (e instanceof BindException) {
+                final StringBuilder sb = new StringBuilder().append(e.getMessage());
+                final InetSocketAddress socketAddress = binding.getValue().getSocketAddress();
+                if (socketAddress != null)
+                    sb.append(" ").append(socketAddress);
+                throw new StartException(sb.toString());
+            } else {
+                throw new StartException("Could not start http listener", e);
```

Comment:
This could use some localization. Even original code should have it.
Looking at it, also message is wrong, as this code handles all listeners not only http ones.
Could you look into this?